src/core/reaction_ensemble.cpp

```diff
@@ -1783,7 +1795,7 @@ double ConstantpHEnsemble::calculate_acceptance_probability(
   return bf;
 }
 
-double WidomInsertion::measure_excess_chemical_potential(int reaction_id) {
+std::vector<double> WidomInsertion::measure_excess_chemical_potential(int reaction_id) {
```

Comment:
Please don't use heap vectors if you know the size in advance.

Reply:
How would i expose a std::array<double,2> in cython?